tests/ttnn/integration_tests/mnist/test_mnist.py

```diff
@@ -0,0 +1,43 @@
+# SPDX-FileCopyrightText: © 2023 Tenstorrent Inc.
+
+# SPDX-License-Identifier: Apache-2.0
+
+import torch
+import ttnn
+import pytest
+from tests.ttnn.utils_for_testing import assert_with_pcc
+from ttnn.model_preprocessing import preprocess_model_parameters
+from models.experimental.mnist.reference.mnist import MnistModel
+from models.experimental.functional_mnist.tt import tt_functional_mnist
+from torch.utils.data import DataLoader
+from torchvision import transforms, datasets
+
+
```

Comment:
Please fast dispatch nightly (which is not very stable) to ensure that your test runs and passes on all 3 card types